### ISSUE TITLE
Extra local:property when user does not specify has_property

### ISSUE THREAD

**Issue by CiaranWelsh** [sys-bio/libOmexMeta#13]
```

cellml = """
<model xmlns="http://www.cellml.org/cellml/1.1#" xmlns:cmeta="http://www.cellml.org/metadata/1.0#"
      name="annotation_examples" cmeta:id="annExamples">
  <component name="main">
    <variable cmeta:id="main.Volume" initial_value="100" name="Volume" units="dimensionless" />
    <variable cmeta:id="main.MembraneVoltage" initial_value="-80" name="MembraneVoltage" units="dimensionless" />
    <variable cmeta:id="main.ReactionRate" initial_value="1" name="ReactionRate" units="dimensionless" />
  </component>
</model>
"""

rdf = RDF()
rdf.set_archive_uri("physical_process.omex")
rdf.set_model_uri("model.cellml")
annot_editor = rdf.to_editor(cellml, generate_new_metaids=False, sbml_semantic_extraction=False)

# fma:14515 = hepatocyte
# fma:66836 = portion of cytosol
with annot_editor.new_physical_entity() as cytosol:
    cytosol \
        .about("cytosol", eUriType.LOCAL_URI) \
        .identity("FMA:66836") \
        .is_part_of("FMA:14515")
    
# http://purl.obolibrary.org/obo/PR_000003767 = alcohol dehydrogenase 1A
with annot_editor.new_physical_entity() as mediator:
    mediator \
        .about("mediator", eUriType.LOCAL_URI) \
        .identity("http://purl.obolibrary.org/obo/PR_000003767") \
        .is_part_of("http://omex-library.org/physical_process.omex/model.rdf#cytosol")
    # \
    #keyword arguments to make sure correct overload is chosen
    #    .has_property("protein", eUriType.LOCAL_URI)

# with annot_editor.new_physical_entity() as source:
#     source.about("source-0", eUriType.LOCAL_URI) \
#                    .identity("http://purl.obolibrary.org/obo/PR_000003767") \
#                    .is_part_of("fma:66836") \
#                    .is_part_of("fma:14515")

# CHEBI:16236 = ethanol
with annot_editor.new_physical_entity() as source:
    source \
        .about("source", eUriType.LOCAL_URI) \
        .identity("CHEBI:16236") \
        .is_part_of("http://omex-library.org/physical_process.omex/model.rdf#cytosol")

# CHEBI:15343 = acetaldehyde
with annot_editor.new_physical_entity() as sink:
    sink \
        .about("sink", eUriType.LOCAL_URI) \
        .identity("CHEBI:15343") \
        .is_part_of("http://omex-library.org/physical_process.omex/model.rdf#cytosol")

# opb:OPB_00592 = chemical molar flow rate
# GO:0004022 = alcohol dehydrogenase (NAD+) activity
with annot_editor.new_physical_process() as reaction_rate:
    reaction_rate \
        .about("process-0", eUriType.LOCAL_URI) \
        .is_version_of("GO:0004022") \
        .add_source("source", eUriType.LOCAL_URI, multiplier=1) \
        .add_sink("sink", eUriType.LOCAL_URI, multiplier=1) \
        .add_mediator("mediator", eUriType.LOCAL_URI) \
        .has_property("main.ReactionRate", eUriType.MODEL_URI, "opb:OPB_00592")

print(rdf)
rdf.draw('tmp')
from IPython.display import IFrame
IFrame("tmp.pdf", width=1024, height=720)

```

produces:

@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
@prefix bqbiol: <http://biomodels.net/biology-qualifiers/> .
@prefix semsim: <http://bime.uw.edu/semsim/> .
@prefix OMEXlib: <http://omex-library.org/> .
@prefix myOMEX: <http://omex-library.org/physical_process.omex/> .
@prefix local: <http://omex-library.org/physical_process.omex/model.rdf#> .

<http://omex-library.org/physical_process.omex/model.cellml#main.ReactionRate>
    bqbiol:isPropertyOf local:process-0 ;
    bqbiol:isVersionOf <https://identifiers.org/opb:OPB_00592> .

local:MediatorParticipant0000
    semsim:hasPhysicalEntityReference local:mediator .

local:Property0000
    bqbiol:isPropertyOf local:cytosol, local:mediator, local:sink, local:source .

local:SinkParticipant0000
    semsim:hasMultiplier "1"^^rdf:double ;
    semsim:hasPhysicalEntityReference local:sink .

local:SourceParticipant0000
    semsim:hasMultiplier "1"^^rdf:double ;
    semsim:hasPhysicalEntityReference local:source .

local:cytosol
    bqbiol:is <https://identifiers.org/FMA:66836> ;
    bqbiol:isPartOf <https://identifiers.org/FMA:14515> .

local:mediator
    bqbiol:is <http://purl.obolibrary.org/obo/PR_000003767> ;
    bqbiol:isPartOf local:cytosol .

local:process-0
    semsim:hasMediatorParticipant local:MediatorParticipant0000 ;
    semsim:hasSinkParticipant local:SinkParticipant0000 ;
    semsim:hasSourceParticipant local:SourceParticipant0000 ;
    bqbiol:isVersionOf <https://identifiers.org/GO:0004022> .

local:sink
    bqbiol:is <https://identifiers.org/CHEBI:15343> ;
    bqbiol:isPartOf local:cytosol .

local:source
    bqbiol:is <https://identifiers.org/CHEBI:16236> ;
    bqbiol:isPartOf local:cytosol .
```


Note the extra Property0000 annotation. This is because user doesn't specify the has_property argument.

**Comment by CiaranWelsh** [comment]
When the call to `hasProperty` is missing, `PropertyBearer` types use the superclass implementation of `property_metaid_base`, which notably doesn't get incremented like other identifiers. It turns out that there was a rogue "#" in the ID generation mechanism that was preventing the exact matches needed for incrementing an ID. 

Fixed in #16

**Comment by CiaranWelsh** [comment]
```
with annot_editor.new_physical_entity() as cytosol:
    cytosol \
        .about("cytosol", eUriType.LOCAL_URI) \
        .identity("FMA:66836") \
        .is_part_of("FMA:14515")
```

produces 

```
local:EntityProperty0000
    bqbiol:isPropertyOf local:cytosol .

local:cytosol
    bqbiol:is <https://identifiers.org/FMA:66836> ;
    bqbiol:isPartOf <https://identifiers.org/FMA:14515> .
```

But should produce 
```
local:cytosol
    bqbiol:is <https://identifiers.org/FMA:66836> ;
    bqbiol:isPartOf <https://identifiers.org/FMA:14515> .
```

**Comment by CiaranWelsh** [comment]
Fixed by #18
